mokuro/script.js

```diff
@@ -119,6 +121,39 @@ function updateProperties() {
         r.style.setProperty('--textBoxDisplay', 'none');
     }
 
+    if (state.toggleOCRTextBoxes) {
+        // Force a hovered state on the clicked .textBox, until the .textBox is clicked off of.
+        let textBoxes = document.querySelectorAll('.textBox');
+        for (let i = 0; i < textBoxes.length; i++) {
+            textBoxes[i].addEventListener('click', function (e) {
+                if (state.toggleOCRTextBoxes) {
+                    if (this.classList.contains('hovered')) {
+                        this.classList.remove('hovered');
```

Comment:
I don't know if that matters for your desired use case with 10ten, but in general I think it'd be more convenient to only remove the hovered class when you click outside the box.